helm/nifi-cluster/Chart.yaml

```diff
@@ -30,7 +30,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 1.1.1
+version: 1.1.2
```

Comment:
Whoops this should actually match the `nifikop/Chart.yaml` version. I think it's a mistake that these have diverged. Right now, everything is versioned together.

Reply:
Ok. I will set it to 1.6.0 same as nifikop/Chart.yaml